doc/rules/alias/no_alias_functions.rst

```diff
@@ -20,24 +20,23 @@ Configuration
 
 List of sets to fix. Defined sets are:
 
-* ``@all`` (all listed sets)
-* ``@internal`` (native functions)
-* ``@exif`` (EXIF functions)
-* ``@ftp`` (FTP functions)
-* ``@IMAP`` (IMAP functions)
-* ``@ldap`` (LDAP functions)
-* ``@mbreg`` (from ``ext-mbstring``)
-* ``@mysqli`` (mysqli functions)
-* ``@oci`` (oci functions)
-* ``@odbc`` (odbc functions)
-* ``@openssl`` (openssl functions)
-* ``@pcntl`` (PCNTL functions)
-* ``@pg`` (pg functions)
-* ``@posix`` (POSIX functions)
-* ``@snmp`` (SNMP functions)
-* ``@sodium`` (libsodium functions)
-* ``@time`` (time functions)
-
+* ``@all`` (all listed sets),
+* ``@internal`` (native functions),
+* ``@exif`` (EXIF functions),
+* ``@ftp`` (FTP functions),
+* ``@IMAP`` (IMAP functions),
+* ``@ldap`` (LDAP functions),
+* ``@mbreg`` (from ``ext-mbstring``),
+* ``@mysqli`` (mysqli functions),
+* ``@oci`` (oci functions),
+* ``@odbc`` (odbc functions),
+* ``@openssl`` (openssl functions),
+* ``@pcntl`` (PCNTL functions),
+* ``@pg`` (pg functions),
+* ``@posix`` (POSIX functions),
+* ``@snmp`` (SNMP functions),
+* ``@sodium`` (libsodium functions),
+* ``@time`` (time functions).
```

Comment:
open question: is it common to put `,` and `.` in bullet list in English?

Reply:
common or not, don't you agree that this change is better than having an exception for this single option?

Reply:
I believe the better option is to keep the proper punctuation for documentation, whatever it takes for code generating it.

Hence, I'm wondering of the proper punctuation in English, for such bullet list.

Reply:
if putting trust into https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/ , it suggest no punctuation or usage of `;`+`.` (instead of `,`+`.`)

Reply:
If you trust that random website, then I do too, updated to `;` + `.`.

Reply:
I admitted i'm very insecure about eng grammar and open for someone who knows better 😅